apps/meteor/app/api/server/v1/rooms.ts

```diff
@@ -581,12 +577,8 @@ API.v1.addRoute(
 				const { dateFrom, dateTo } = this.bodyParams;
 				const { format } = this.bodyParams;
 
-				if (!['html', 'json'].includes(format || '')) {
-					throw new Meteor.Error('error-invalid-format');
-				}
-
-				const convertedDateFrom = new Date(dateFrom || '');
-				const convertedDateTo = new Date(dateTo || '');
+				const convertedDateFrom = dateFrom ? new Date(dateFrom) : new Date(0);
```

Comment:
This made me curious too, but turns out `coerceTypes` will cast the data to `string`, since this is the `type` of this param. So we still need to convert it into a date.